2022-08-31 20:05:01,769 ERROR com.bankingservice.banking.exception.ApplicationExceptionHandler [http-nio-9191-exec-3] [d766ab0b-64e6-472c-95bd-746590dde2f9] [067f2beb-d0b3-4b87-a708-955eb1f482c5] UserNotFoundException: [Ljava.lang.StackTraceElement;@6ff8d8c
2022-08-31 20:26:18,272 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [getUserDetails] user does not exist for user ID : PEZVzItBWL
2022-08-31 20:26:29,327 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [getUserDetails] user does not exist for user ID : meIeVtpZcS
2022-08-31 20:26:40,735 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [generateCardDetails] card already exists for userId: T2l2B3RXZG
2022-08-31 20:26:40,933 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [generateCardDetails] user does not exist with user ID: T2l2B3RXZG
2022-08-31 20:26:41,520 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [getCardDetails] card does not exist for user ID : T2l2B3RXZG
2022-08-31 20:26:41,670 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [getCardDetails] user does not exist for user ID : T2l2B3RXZG
2022-08-31 20:26:42,176 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [getUserDetails] user does not exist for user ID : T2l2B3RXZG
2022-08-31 20:26:42,926 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [saveRegisterModel] account already exists for userId: T2l2B3RXZG
2022-08-31 20:26:43,538 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [insertDetailsForRegistration] details insertion failed in db for request: RegisterRequestDTO(name=qguscILhTg, email=xCHgOchzZP, mobileNumber=2685187617609516398, userName=pxgLcoWUmt, password=lOJPtCTMyN)
org.springframework.dao.DataIntegrityViolationException: 
	at mockit.internal.expectations.invocation.InvocationResult$ThrowableResult.produceResult(InvocationResult.java:41)
	at mockit.internal.expectations.invocation.InvocationResult.produceResult(InvocationResult.java:22)
	at mockit.internal.expectations.invocation.InvocationResults.produceResult(InvocationResults.java:124)
	at mockit.internal.expectations.Expectation.produceResult(Expectation.java:48)
	at mockit.internal.expectations.ReplayPhase.produceResult(ReplayPhase.java:108)
	at mockit.internal.expectations.ReplayPhase.handleInvocation(ReplayPhase.java:52)
	at mockit.internal.expectations.RecordAndReplayExecution.recordOrReplay(RecordAndReplayExecution.java:170)
	at com.bankingservice.banking.services.servicehelper.AccountServiceHelper.saveRegisterModel(AccountServiceHelper.java)
	at com.bankingservice.banking.services.AccountService.insertDetailsForRegistration(AccountService.java:48)
	at com.bankingservice.banking.services.AccountServiceTest.testInsertDetailsForRegistrationException(AccountServiceTest.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
2022-08-31 20:26:44,024 ERROR com.bankingservice.banking.services.AccountService [main] [] [] [setPin] card not found for card number: 4038263027211082284
